app/models/patient.rb

```diff
@@ -609,7 +593,7 @@ def end_of_monitoring
 
   # Date when patient is expected to be purged
   def expected_purge_date
-    (updated_at + ADMIN_OPTIONS['purgeable_after'].minutes)&.rfc2822
+    (updated_at + ADMIN_OPTIONS['purgeable_after'].minutes)&.rfc2822 unless monitoring
```

Comment:
I think I might vaguely remember @holmesie asking me to add this here

Reply:
I don't actually recall asking for this, however it does make sense to do, particularly because this is included in things like the csv linelist export (and it doesn't make sense to include a date if the record is active). To address Kylie's valid concern, perhaps it should also have a default return if monitoring is inactive (e.g. `Active Monitoring`).

Reply:
how about this instead?

```
def expected_purge_date
  monitoring ? (updated_at + ADMIN_OPTIONS['purgeable_after'].minutes)&.rfc2822 : ''
end
```

Reply:
https://github.com/SaraAlert/SaraAlert/pull/562/commits/18f2cd1c1295b23fca3eb469bf542092217c6acb#diff-94dcc014f62f0d9b6084ad98f82f74cd40c86b029ef1691914bdbc960c2f42dcR596

Reply:
I think that works!

Reply:
So that change makes it so if they are closed, it doesn't show an expected purge date. I think we may want that flipped.

Reply:
https://github.com/SaraAlert/SaraAlert/pull/562/commits/6c82b3a778c6f7bf59322cd7b63f99706a0229a9#diff-94dcc014f62f0d9b6084ad98f82f74cd40c86b029ef1691914bdbc960c2f42dcR596